webhook-service/go.mod

```diff
@@ -16,45 +17,78 @@ require (
 )
 
 require (
+	cloud.google.com/go v0.54.0 // indirect
+	github.com/Azure/go-autorest v14.2.0+incompatible // indirect
```

Comment:
that's quite a lot of new dependencies - are they really required, or can this be tidied up using go mod tidy?